src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -6423,6 +6423,97 @@ def test_aks_create_with_standard_csi_drivers(self, resource_group, resource_gro
             self.is_empty(),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='centraluseuap', preserve_default_location=True)
+    def test_aks_create_with_standard_blob_csi_driver(self, resource_group, resource_group_location):
```

Comment:
Please upload the recording yaml files for those new tests. Otherwise, CI tasks will fail due to missing yaml file